src/Magnum/Timeline.h

```diff
@@ -134,6 +134,8 @@ class MAGNUM_EXPORT Timeline {
          */
         Float previousFrameDuration() const { return _previousFrameDuration; }
 
+        float currentFrameDuration() const;
```

Comment:
```suggestion
        Float currentFrameDuration() const;
```

And a documentation block wouldn't hurt, especially clarifying the behavior for what it does when the timeline is stopped -- what should it return in that case? 0? I'm thinking it probably shouldn't return some nasty huge number.